assets/src/blocks/Timeline/TimelineBlock.js

```diff
@@ -1,126 +1,58 @@
-import {Timeline} from './Timeline.js';
+import {frontendRendered} from '../frontendRendered';
+import {TimelineEditor} from './TimelineEditor';
 
 const {__} = wp.i18n;
+const BLOCK_NAME = 'planet4-blocks/timeline';
+
+const attributes = {
+  timeline_title: {
+    type: 'string',
+    default: ''
+  },
+  description: {
+    type: 'string',
+    default: ''
+  },
+  google_sheets_url: {
+    type: 'string',
+    default: ''
+  },
+  language: {
+    type: 'string',
+    default: 'en',
+  },
+  timenav_position: {
+    type: 'string',
+    default: ''
+  },
+  start_at_end: {
+    type: 'boolean',
+    default: false
+  },
+};
 
 export class TimelineBlock {
```

Comment:
Maybe we should start changing this too while we doing away with classes in favor of hooks. These classes are just a shell around the `registerBlockType` function, where we never use an instance of the class and just call the constructor for a side effect in it. Just a function which has whatever happens in the constructor seems a lot more straightforward.

Reply:
Indeed, initially we thought these would be classes but in the end it was never the case, I'll make this change here and create a separate PR to change all the other ones and remove those pointless `new BlockName()...` instantiations because I know you despise them! 🔥